src/driver/cmd.rs

```diff
@@ -219,6 +226,12 @@ pub enum CollectorName {
     SweepSwiper,
 }
 
+#[derive(Copy, Clone, Debug, RustcDecodable)]
+pub enum BaselineName {
+    Cannon,
+    Standard,
```

Comment:
We should find a name for the old baseline compiler that we can keep even when this compiler isn't the default one anymore.

Reply:
I don't have any good idea for a themed name. In the code the "standard" baseline compiler is simply called AstCodeGen. We could that name everywhere.